### PR TITLE
Jn adventure

### PR DESCRIPTION
**Pull request by jackngai**
Add snowman story line

### CODE REVIEW

**Comment by jackngai** [comment]
Hey Casey, can you review this? @casey-collab

File: english/snowman/ice_monster.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Furious for being rejected, the Ice Queen created the Ice Monster!
```
Comment (casey-collab) [inline english/snowman/ice_monster.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: english/snowman/olaf.md [hunk 2]
```diff
@@ -0,0 +1 @@
+The Ice Queen created Olaf!
```
Comment (casey-collab) [inline english/snowman/olaf.md]: This file has also been added to a directory that already contained a page.

File: english/snowman/snowman.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+[Yes] (olaf.md)
```
Comment (casey-collab) [inline english/snowman/snowman.md]: This file has also been added to a directory that already contained a page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/snowman/ice_monster.md (hunk 1 above)
Comment (jackngai) [inline english/snowman/ice_monster.md]: Corrected.

File: english/snowman/olaf.md (hunk 2 above)
Comment (jackngai) [inline english/snowman/olaf.md]: Corrected.

**Comment by jackngai** [comment]
@casey-collab Please take a look.

File: english/snowman/ice_monster.md (hunk 1 above)
Comment (casey-collab) [inline english/snowman/ice_monster.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: english/snowman/olaf.md (hunk 2 above)
Comment (casey-collab) [inline english/snowman/olaf.md]: This file has also been added to a directory that already contained a page.

File: english/snowman/snowman.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+[Yes] (olaf/olaf.md)
```
Comment (casey-collab) [inline english/snowman/snowman.md]: This file has also been added to a directory that already contained a page.

File: english/snowman/olaf.md (hunk 2 above)
Comment (casey-collab) [inline english/snowman/olaf.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

File: english/snowman/ice_monster.md (hunk 1 above)
Comment (casey-collab) [inline english/snowman/ice_monster.md]: This page is also unreachable.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by jackngai** [comment]
Please check again, @casey-collab.

**Comment by casey-collab** [comment]
Looks great! :sunny:

**Pull request merged**